File: support_tools/swigwin-3.0.12/Examples/test-suite/csharp/rename_pcre_encoder_runme.cs
using System;
using rename_pcre_encoderNamespace;

public class runme {
  static void Main() {
    SomeWidget w = new SomeWidget();
    w.put_borderWidth(17);
    if ( w.get_borderWidth() != 17 )
      throw new Exception(String.Format("Border with should be 17, not {0}",
                                        w.get_borderWidth()));

    if ( rename_pcre_encoder.StartINSAneAndUNSAvoryTraNSAtlanticRaNSAck() != 42 )
      throw new Exception("Unexpected result of renamed function call");
  }
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/rename_pcre_encoder_runme.py
from rename_pcre_encoder import *

s = SomeWidget()
s.put_borderWidth(3)
if s.get_borderWidth() != 3:
    raise RuntimeError("Border should be 3, not %d" % (s.get_borderWidth(),))

s.put_size(4, 5)
a = AnotherWidget()
a.DoSomething()

evt = wxEVTSomeEvent()
t = xUnchangedName()

if StartINSAneAndUNSAvoryTraNSAtlanticRaNSAck() != 42:
    raise RuntimeError("Unexpected result of renamed function call")
